<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>output</class>
 <widget class="QDialog" name="output">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <widget class="QPushButton" name="dfsBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обход в глубину</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bfsBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Обход в ширину</string>
   </property>
  </widget>
  <widget class="QPushButton" name="floydBtn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Алгоритм Флойда</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dijkstraBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Алгоритмт
Дeйкстры</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>200</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Стартовая позиция</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>620</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Результат поиска</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="outputDist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>830</width>
     <height>450</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
